CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [1.34](https://github.com/brave/brave-ios/releases/tag/v1.34)
+
+- Added support for initial set of Brave Widgets. ([#4329](https://github.com/brave/brave-ios/issues/4329))
+- Implemented "Login & Passwords" feature for managing saved credentials. ([#4112](https://github.com/brave/brave-ios/issues/4112))
+- [Security] Fixed reader mode issue as reported on HackerOne by nishimunea. ([#4772](https://github.com/brave/brave-ios/issues/4772))
+- [Security] Fixed message token handling as reported on HackerOne by nishimunea. ([#4775](https://github.com/brave/brave-ios/issues/4775))
+- [Security] Added "no-referrer" for content for all internal pages as reported on HackerOne by nishimunea. ([#4784](https://github.com/brave/brave-ios/issues/4784))
+- Removed Canvas Fingerprinting protection. ([#4522](https://github.com/brave/brave-ios/issues/4522))
```

Comment:
I know we usually don't put `Removed` as the top entry, but maybe we should this time around so it's easier for users to spot the change? If not at the top, probably the second one right under widgets.

Reply:
I think @anthonypkeane should call this out in the AppStore release notes as well and not just here. Since this is something we are removing it should be made aware on the AppStore notes as well. I'll move it to be second on the list

Reply:
Agreed 👍 Mentioned it in the sign off thread via `#release` just in case.